src/components/Road.vue

```diff
@@ -91,6 +91,19 @@ export default {
         this.$emit('change-year', newYear);
       }
     }
+  },
+  watch: {
+    visibleList: function (newVisibleList) {
+      if (this.$store.state.cookiesAllowed) {
+        this.$cookies.set('visibleList' + this.roadID, JSON.stringify(newVisibleList));
+      }
+    }
+  },
+  mounted () {
+    const visibleList = JSON.parse(this.$cookies.get('visibleList' + this.roadID));
+    if (this.$store.state.cookiesAllowed && visibleList) {
+      this.visibleList = visibleList;
+    };
```

Comment:
which PR do we need to merge first? these are a totally separate feature it seems?

Reply:
Yeah, this is unrelated.  Result of a bad merge after https://github.com/sipb/courseroad2/commit/314f09f76869309d828fcaf7d4dc3a53b380f117 went in to master it seems